frontend-react/e2e/spec/chromium-only/public-pages-link-check.spec.ts

```diff
@@ -7,7 +7,7 @@ import * as publicPagesLinkCheck from "../../pages/public-pages-link-check";
 // Since we're just checking link validity. This is specified within our
 // Playwright.config here:
 // {
-//   name: "chromium-specific",
+//   name: "chromium-only",
```

Comment:
should this be uncommented?

Reply:
no, this is just an in-depth comment of what we're doing here and why this is a special case:

https://github.com/CDCgov/prime-reportstream/blob/8e84192b467bbc606447f8861d42606b1fbcac49/frontend-react/e2e/spec/chromium-only/public-pages-link-check.spec.ts#L1-L14